### PR TITLE
Outputter Added

### PR DESCRIPTION
**Pull request by olhabn**

### CODE REVIEW

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            name: 'outputValue',
+            code: function(message) {
+                var SBF = '';
+                //MTn96 and MTn92 lacks a fieldtag at the last position, so translating leads to crash
+                for (var i = 1; i < message.length + 1; i++)
+                {
+                    //note, the i is NOT correct for the below command, modification required
+                    var field = foam.lookup("net.nanopay.swift.fields.FieldTag" + i).create();
+                    SBF += field.toSBF(message);
+                }
```
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:51]: I'd rather do
`message.cls_.getAxiomsByClass(net.nanopay.swift.fields.FieldTagRegexValidated).forEach(p => SBF += p.toSBF(message));`

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            required: true
+        },
+    ],
+
+    methods: [
+        {
+            name: 'outputValue',
+            code: function(message) {
+                var SBF = '';
+                //MTn96 and MTn92 lacks a fieldtag at the last position, so translating leads to crash
```
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:45]: do we still have this problem?

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            name: 'of',
+            required: true
+        },
+    ],
+
+    methods: [
+        {
+            name: 'outputValue',
+            code: function(message) {
+                var SBF = '';
```
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:44]: now as we have generateBasicHeader() and generateApplicationHeaderInput()(hopefully)  can you suggest how to make our output to look exactly like example in Parsing a [SWIFT Message document](https://www.immagic.com/eLibrary/ARCHIVES/GENERAL/IONA_IE/I070510D.pdf) page 2?

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                message.cls_.getAxiomsByClass(net.nanopay.swift.fields.FieldTagRegexValidated).forEach(p => SBF += p.toSBF(message));
+                return SBF;
+            },
+        },
```
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:48]: can you add `outputFObject` method as in `foam3/src/foam/lib/csv/CSVOutputterImpl.js`?
and now as we have generateBasicHeader() and generateApplicationHeaderInput()(hopefully)  can you suggest how to make our output to look exactly like example in Parsing a [SWIFT Message document](https://www.immagic.com/eLibrary/ARCHIVES/GENERAL/IONA_IE/I070510D.pdf) page 2?

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            class: 'Class',
+            name: 'of',
+            required: true
+        },
+    ],
+
+    methods: [
+        {
+            name: 'outputValue',
```
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:42]: I'd rather name it outputFObject for now

File: src/foam/swift/SWIFTOutputterImpl.js (hunk 3 above)
Comment (olhabn) [inline src/foam/swift/SWIFTOutputterImpl.js:44]: and now as we have generateBasicHeader() and generateApplicationHeaderInput()(hopefully)  can you suggest how to make our output to look exactly like example in Parsing a [SWIFT Message document](https://www.immagic.com/eLibrary/ARCHIVES/GENERAL/IONA_IE/I070510D.pdf) page 2?

File: src/foam/parse/FScriptTest.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/parse/FScriptTest.js:3]: 2021

File: src/foam/swift/SWIFTOutputterImpl.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+ * The intellectual and technical concepts contained
+ * herein are proprietary to nanopay Corporation
+ * and may be covered by Canadian and Foreign Patents, patents
+ * in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from nanopay Corporation.
+ */
+
+foam.CLASS({
```
Comment (kgrgreer) [inline src/foam/swift/SWIFTOutputterImpl.js]: indentation should be 2 spaces

**Pull request closed without merging**